src/main/java/com/oracle/nosql/spring/data/core/mapping/NosqlTable.java

```diff
@@ -76,6 +103,13 @@
      */
     String consistency() default Constants.DEFAULT_TABLE_CONSISTENCY;
 
+    /**
+     * Sets the default durability for all write operations applied to this
+     * table. Valid values for this are defined in {@link Durability}.
+     * If not set the default value for this is COMMIT_NO_SYNC.
+     */
+    String durability() default Constants.DEFAULT_TABLE_CONSISTENCY;
```

Comment:
consistency? shouldn't this be DURABILITY?

Reply:
Correct.